docs/reference/cluster_manifest.md

```diff
@@ -175,6 +175,9 @@ explanation of `ttl` and `loop_wait` parameters.
 * **slots**
   permanent replication slots that Patroni preserves after failover by re-creating them on the new primary immediately after doing a promote. Slots could be reconfigured with the help of `patronictl edit-config`. It is the responsibility of a user to avoid clashes in names between replication slots automatically created by Patroni for cluster members and permanent replication slots. Optional.
 
+* **standby_cluster**
```

Comment:
please also extend the `manifests/complete-postgres-manifest.yaml` with a commented-out example of how this new field can be used

Reply:
Done.